### ISSUE TITLE
Approval requirement not being respected

### ISSUE THREAD

**Issue by veritas2884** [tidusjar/Ombi#950]
#### Ombi Version:

V 2.0.1234
#### Update Branch:

Stable

#### Operating System:

Mono / Ubuntu 16.0.4

#### Problem Description:
I have  Settings > User Management > Auto Approve TV Show Requests  unchecked

However, TV show requests are being automatically piped to sonarr without my prior approval. This issue did not happen in the last version of PR.NET

#### Reproduction Steps:

Every show that is added by a user is not requiring approval. 

Thank you for any help!

**Comment by veritas2884** [comment]
I did some more digging, under the main page User Management page, where you can see individual users, they all have the AutoApproveTv permission. I can remove it if I edit one by one, but checking and unchecking the global setting ( Settings > User Management > Auto Approve TV Show Requests ) does not remove it from all users.

**Comment by SuperPotatoMen** [comment]
That is because the global settings are not realtime. Wait for the job has run and check again ;)

**Comment by tidusjar** [comment]
The global settings are only applied when you share your server with a new
Plex user. There is a request to bulk modify

**Comment by veritas2884** [comment]
Thank you both for the replies. The issue is that it defaulted to auto approving TV shows. When I setup ombi, I selected that I did not want TV shows approved automatically. With 30+ users on my plex server it makes it a long process to fix this one by one. If I blow away the install and redo the setup, is there a way to prevent having to manually modify each user? Also, thank you so much for continuing to develop this awesome program. It is fantastic.
